pypredict/requirements.txt

```diff
@@ -1,6 +1,6 @@
 fastapi
 jinja2
-pyensign==0.7b0
-river>=0.17.0
+pyensign>=0.7b1
+river==0.15.0
```

Comment:
After playing around with installing rust and failing, I discovered that using version 0.15.0 of river solved the problem on my machine.  This is less than ideal so if you are able to create a container with the latest version, I am open to reverting this back to the latest version and we can add a comment in the readme for anyone who runs into the same problem that I had.

Reply:
there were a few new utilities for preprocessing, metrics, feature extraction and some other functionalities, which would be worthwhile upgrading for.  I am curious if you are able to build using the latest version and if so, can you let me know what docker version you are using?

Reply:
I was able to build with 0.17.0, my version of docker is:

Docker version 20.10.23, build 7155243

Reply:
there must be something specific to my machine - I updated to your version and I am still running into the issue.

Reply:
It's up to you but I think we can just upgrade to the latest version of river since it works on your machine.

Reply:
also, it appears that introducing `[metrics.RollingROCAUC](https://riverml.xyz/0.16.0/api/metrics/RollingROCAUC).` is what may be causing the issue.  It requires a rust compiler.

Reply:
Alright, I can update the river version from my side, merge the PR and deploy.